airbase/fetch.py

```diff
@@ -0,0 +1,274 @@
+"""Helper functions encapsulating async HTTP request and file IO"""
+from __future__ import annotations
+
+import asyncio
+import json
+import sys
+import warnings
+from pathlib import Path
+from turtle import down
```

Comment:
This is unused and raises a `ModuleNotFound` error on my system. Added by accident?

```pycon
>>> import airbase
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/cw0504/personal/airbase/airbase/__init__.py", line 2, in <module>
    from .airbase import AirbaseClient, AirbaseRequest
  File "/Users/cw0504/personal/airbase/airbase/airbase.py", line 7, in <module>
    from .fetch import (
  File "/Users/cw0504/personal/airbase/airbase/fetch.py", line 9, in <module>
    from turtle import down
  File "/Users/cw0504/.pyenv/versions/3.7.1/lib/python3.7/turtle.py", line 107, in <module>
    import tkinter as TK
  File "/Users/cw0504/.pyenv/versions/3.7.1/lib/python3.7/tkinter/__init__.py", line 36, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
ModuleNotFoundError: No module named '_tkinter'
```

Reply:
mmmm, that was probably VS code auto-import.
Will remove it